alcs-frontend/src/app/services/planning-review/planning-review-decision/planning-review-decision.service.spec.ts

```diff
@@ -62,6 +62,34 @@ describe('PlanningReviewDecisionService', () => {
     expect(toastService.showErrorToast).toHaveBeenCalledTimes(1);
   });
 
+  it('should fetch and return codes', async () => {
+    httpClient.get.mockReturnValue(
+      of([
+        {
+          fileNumber: '1',
+        },
+      ]),
+    );
+
+    const res = await service.fetchCodes();
+
+    expect(res).toBeDefined();
+    expect(httpClient.get).toHaveBeenCalledTimes(1);
```

Comment:
Seems like it doesn't check if a code has been returned, and the mock seems to return a file number. Is that what is meant by code?